src/components/MultiStep/index.ts

```diff
@@ -147,37 +147,38 @@ class MultiStep extends BridgeBase {
     return sStyle;
   }
 
-  getStepHorizontallMarkerStyle(step: any, bSubStep: boolean = false): string {
+  getStepHorizontallMarkerStyle(step: any, type): string {
     let sStyle = '';
-    switch (step.visited_status) {
-      case 'success':
-        if (bSubStep) {
-          sStyle = 'psdk-horizontal-marker h-success h-sub';
+    switch (type) {
+      case 'label':
+        if (step.visited_status === 'current') {
+          sStyle = 'psdk-horizontal-step-label-selected';
         } else {
-          sStyle = 'psdk-horizontal-marker h-success';
+          sStyle = 'psdk-horizontal-step-label';
         }
         break;
-      case 'current':
-        if (bSubStep) {
-          sStyle = 'psdk-horizontal-marker h-current h-sub';
+      case 'icon':
+        if (step.visited_status === 'current') {
+          sStyle = 'psdk-horizontal-step-icon-selected';
         } else {
-          sStyle = 'psdk-horizontal-marker h-current';
-        }
-        break;
-      case 'future':
-        if (bSubStep) {
-          sStyle = 'psdk-horizontal-marker h-future h-sub';
-        } else {
-          sStyle = 'psdk-horizontal-marker h-future';
+          sStyle = 'psdk-horizontal-step-icon';
         }
         break;
       default:
         break;
     }
-
     return sStyle;
   }
 
+  _showHLine(index: number): boolean {
+    // eslint-disable-next-line sonarjs/prefer-single-boolean-return
+    if (index < this.arNavigationSteps.length - 1) {
```

Comment:
Can change this method to: `return index < this.arNavigationSteps.length - 1` to be more leaner.